automatic_spinnaker_make.sh

```diff
@@ -0,0 +1,29 @@
+# This script assumes it is run from the directory holding all github projects in parellel
+# sh SupportScripts/automatic_make.sh
+
+cd spinnaker_tools
+source $PWD/setup
+make clean
+make || exit $?
+cd ..
+cd spinn_common
+make clean
+make || exit $?
+make install
+cd ..
+cd SpiNNFrontEndCommon/c_common/
+cd front_end_common_lib/
+make install-clean
+cd ..
+make clean
+make || exit $?
+make install
+cd ../..
+cd sPyNNaker/neural_modelling/
+make clean
+make || exit $?
+source $PWD/setup
```

Comment:
Lines 22-25 should be:
```
source sPyNNaker/neural_modelling/setup.bash
make -C sPyNNaker/neural_modelling clean all
```
(The above depends on my proposed rename of `setup` which also addresses the really big bugs in it.)